wallet-new/src/Cardano/Wallet/Orphans/Aeson.hs

```diff
@@ -28,7 +28,7 @@ instance FromJSON BackupPhrase where
 instance ToJSON CFilePath where
   toJSON (CFilePath c) = toJSON c
 
-deriveJSON defaultOptions ''Core.Coin
+-- deriveJSON defaultOptions ''Core.Coin
```

Comment:
Why have this been commented out?

Reply:
@adinapoli-iohk please see the last paragraph in the PR description.

Reply:
@Pastafarianist Sorry, my bad! This is what happens by glossing over the PR description 😉 . If the instances are equivalent I'm ok with removing the code from the `Wallet.Orphans.Aeson` module, as I can use the ones in `Pos.Aeson.Core`.

If they are not, we will need to come up with a different strategy. The tension in the design space is that we are trying to design `wallet-new` with backward compatibility in mind, so we ideally don't want neither to modify the `Pos.Aeson.Core` instance to fits `wallet-new` needs nor the other way around.